pages/success-banner/index.js

```diff
@@ -1,23 +1,49 @@
 import React from 'react';
 import Head from 'next/head';
 
+/**
+ * @summary Make and return an event method for a given event description string
+ */
 const eventLog = (eventDesc) => {
   const data = 'Banner ' + eventDesc;
 
   return () => {
     const url = new URL(location.href);
 
-    if (url.searchParams.get('etcher-version').indexOf('1.0.0') !== -1) {
-      console.log(data);
-
-    } else {
+    // Use the 'new' API version for more recent versions of Etcher utilizing
+    // the Robot object format.
+    if (url.searchParams.get('api-version') === '1') {
```

Comment:
Fair enough. I just assumed that the API would only gain features, not lose them ;) But as you say, it's easy enough to change the banner anyway.